server/config.example.py

```diff
@@ -1,10 +1,24 @@
 # Copy this file to config.py
-SQLALCHEMY_DATABASE_URI = 'postgresql+psycopg2://optima:optima@localhost:5432/optima'
-SECRET_KEY = 'F12Zr47j\3yX R~X@H!jmM]Lwf/,?KT'
-UPLOAD_FOLDER = '/tmp/uploads'
-CELERY_BROKER_URL = 'redis://localhost:6379'
-CELERY_RESULT_BACKEND = 'redis://localhost:6379'
-CELERY_ACCEPT_CONTENT = ['pickle', 'json', 'msgpack', 'yaml']
-SQLALCHEMY_TRACK_MODIFICATIONS = False
-REDIS_URL = CELERY_BROKER_URL
-MATPLOTLIB_BACKEND = "agg"
+
+import os
+import multiprocessing
+import math
+
+SQLALCHEMY_DATABASE_URI = os.getenv('SQLALCHEMY_DATABASE_URI','postgresql+psycopg2://optima:optima@localhost:5432/optima')
+SECRET_KEY = os.getenv('SECRET_KEY','F12Zr47j\3yX R~X@H!jmM]Lwf/,?KT')
+UPLOAD_FOLDER = os.getenv('UPLOAD_FOLDER','/tmp/uploads')
+REDIS_URL = os.getenv('REDIS_URL', 'redis://127.0.0.1:6379/4') # Shortcut for setting both the celery broker and result backend cache
+CELERY_BROKER_URL = os.getenv('CELERY_BROKER_URL',REDIS_URL)
+CELERY_RESULT_BACKEND = os.getenv('CELERY_RESULT_BACKEND',REDIS_URL)
+CELERY_ACCEPT_CONTENT = os.getenv('CELERY_ACCEPT_CONTENT','pickle,json,msgpack,yaml').split(',') # Comma separated list
+SQLALCHEMY_TRACK_MODIFICATIONS = bool(os.getenv('SQLALCHEMY_TRACK_MODIFICATIONS',False))
+MATPLOTLIB_BACKEND = os.getenv('MATPLOTLIB_BACKEND',"agg")
+SERVER_PORT = int(os.getenv('PORT', 8080))
+WORKERS = int(os.getenv('WORKERS',min(math.ceil(multiprocessing.cpu_count()/2.0),8))) # By default use half the CPUs, up to a maximum of 8
+
+# GUNICORN VARIABLES
+bind = '0.0.0.0:%d' % (SERVER_PORT) # Bind to 0.0.0.0 so apollo can talk to athena. The port should generally be publicly firewalled
```

Comment:
@RomeshA It seems like this should be something from an environment variable or the systemctl script, not in the Optima HIV repo?

Reply:
Let's just make it a default with an env variable override like the others. I think `0.0.0.0` would be fine under most circumstances. I'll also clarify the comment - it's not actually that specific to athena/apollo, more about the choice of `0.0.0.0` vs `127.0.0.1` depending on whether you want to be able to connect to it from other machines or not